src/runmodes.c

```diff
@@ -982,12 +982,15 @@ float threading_detect_ratio = 1;
  */
 void RunModeInitializeThreadSettings(void)
 {
-    threading_set_cpu_affinity = FALSE;
-    if ((ConfGetBool("threading.set-cpu-affinity", &threading_set_cpu_affinity)) == 0) {
-        threading_set_cpu_affinity = FALSE;
+    int affinity = 0;
+    if ((ConfGetBool("threading.set-cpu-affinity", &affinity)) == 0) {
+        threading_set_cpu_affinity = false;
+    } else {
+        threading_set_cpu_affinity = affinity == 1;
```

Comment:
Is this a better way of doing the same thing in C? Trying to justify the addition of new var here.

Reply:
`ConfGetBool` doesn't return a C `bool` -- thus `affinity` is an integer type that is then used to set the global (to the source module) scoped `bool threading_set_cpu_affinity`